src/components/molecules/InformationCard/InformationCard.tsx

```diff
@@ -15,7 +17,9 @@ const InformationCard: React.FC<InformationCardProps> = ({
   className,
 }) => (
   <div className={classNames("information-card-container", className)}>
-    <h4 className="title">{title}</h4>
+    <h4 className="title">
```

Comment:
Pretty sure rendering in a title tag like this will cause issues similar to the p in p issue

Reply:
didn't see the problem, but indeed decided to not bother about titles (left sample event though with markdown for testing).  Removed them all.  Now this PR has only markdown rendering for event description